src/AbstractEvent.php

```diff
@@ -19,7 +19,7 @@
     protected $emitter;
 
     /**
-     * {@inheritdoc}
+     * @inheritdoc
```

Comment:
What I was trying to get at is that just because phpDocumetor supports that, it doesn't mean everyone else does, and, according to the spec, param/return/throws are inherited anyway, so adding inherit doc does nothing there.

Summary:

Before this change: everything inherited.
After this change: the description is not inherited but everything else is.